@shared/api/internal.ts

```diff
@@ -1176,11 +1176,11 @@ export const saveAllowList = async ({
 export const saveSettings = async ({
   isDataSharingAllowed,
   isMemoValidationEnabled,
-  isNonSSLEnabled,
+  isHideDustEnabled,
```

Comment:
there was a bug where, i think through a bad merge, the `isNonSSLEnabled` configuration was in both the Preferences and in the Advanced Settings. It used to be in Preferences but was later moved to Advanced Settings.

Taking this opportunity to replace errant `isNonSSLEnabled` config with the new `isHideDustEnabled`